standards/8x.md

```diff
@@ -0,0 +1,111 @@
+# Specification
+
+## Summary
+A standardized interface for interacting with recurring subscriptions.
+
+##  Abstract
+This standard defines a contract interface for interacting with subscription data. The 8x suggestions come under the feelings that implementation of how a subscription is created, is left up to the implementor. If we define a standard for implementation, we'll be locking ourselves into one way of doing things.
+
+## Our Thoughts
+The idea behind a standard should be an easy way for developers to read/access data regardless of an implementation. A standard shouldn't govern the architecture of a system. It's for this reason that functions such as adding and editing seem application/protocol specific.
+
+Also features such as editing go against the principal of putting users-first. Just because we have the approval to take tokens from a user's wallet, that doesn't mean they should be uninformed of what happens. In the case of a vendor changing prices, they need to create a new plan, revoke the old one and users then need to explicitly upgrade.
```

Comment:
I left a comment on the other PR about firing Event's when this happens.

I agree with the point in the case of important data like amount, but i think that things like pausing can be done in a user-first way